pansat/catalogue/local.py

```diff
@@ -0,0 +1,148 @@
+"""
+pansat.catalogue.local
+===================================
+This module defines the a catalogue class to look at and list information about downloaded files in local directory.
+
+"""
+
+
+import datetime
+import glob
+import os
+from pathlib import Path
+from pansat.products import ALL_PRODUCTS
+from pansat.products.reanalysis import era5, ncep
+from pansat.products.satellite import cloud_sat, calipso, dardar, gpm
+from pansat.products.ground_based import opera
+
+
+DEFAULT_DESTINATIONS = [
```

Comment:
It would be better to get this information from the products ``default_destination`` attribute.

Reply:
OK, agree. But would it be better then to make the first element of the default destinations class variables maybe? Because all CloudSat products are, for instance, stored in "Cloudsat/product_name/" and what we want here is only the first part ("CloudSat"), which at the moment we can only get when we instantiate the object even though it is the same for all objects of this class